src/cac.ts

```diff
@@ -24,6 +24,10 @@ export default async function tab(
 ) {
   const completion = new Completion();
 
+  // a hidden flag to track if -- is there in the raw arguments? we might need a better way to do this?
```

Comment:
the comment is not needed since we console log a similar error with the same text